berth_reservations/tests/conftest.py

```diff
@@ -38,6 +27,31 @@ def superuser():
     return user
 
 
+@pytest.fixture
+def api_client():
+    return create_api_client()
+
+
+@pytest.fixture
+def user_api_client(user):
+    return create_api_client(user=UserFactory())
+
+
+@pytest.fixture
+def staff_api_client():
```

Comment:
Why do the other two user fixtures receive a user/superuser argument, but not `staff`?

Reply:
good catch! none of them should receive any arguments. this is remnants of my tinkering. 🙂